can-bus/dbcs/CanMsgs.dbc

```diff
@@ -3,29 +3,50 @@ NS_:
 BS_:
 
 
-BU_: BMS DCM DIM FSM GSM INVL INVR PDM DEBUG
+BU_: DIM BMS DCM GSM INVL INVR FSM PDM DEBUG
 
 
-BO_ 100 BMS_Vitals: 1 BMS
-SG_ Heartbeat : 0|1@1+ (1,0) [0|1] "" DEBUG,DCM,FSM,PDM
-SG_ CurrentState : 1|3@1+ (1,0) [0|4] "" DEBUG,DCM,FSM,PDM
-SG_ StateOfCharge : 4|3@1+ (14.285714285714286,0) [0|100] "" DEBUG,DCM,FSM,PDM
+BO_ 502 DIM_Vitals: 1 DIM
+SG_ Heartbeat : 0|1@1+ (1,0) [0|1] "" DEBUG,BMS,DCM
+SG_ State : 1|1@1+ (1,0) [0|0] "" DEBUG,BMS,DCM
```

Comment:
We don't track the DBC anymore (as of today) since it's autogenerated and caused gross merge conflicts, so please delete this too